src/lib/pages/backoffice/Item/ItemDetails/ItemCirculation/ItemCirculation.js

```diff
@@ -23,7 +23,10 @@ import {
 
 class ItemStatusMessageOnLoan extends Component {
   render() {
-    const { circulation } = this.props;
+    const { circulation, title } = this.props;
+
+    const emailBody = `Dear ${circulation.patron.name}%2c </br></br> We are contacting you about your loan on "${title}"</br></br></br> <i>${invenioConfig.APP.emailSignature}</i>`;
+    const emailSubject = `${invenioConfig.APP.emailSignature} Your loan on "${title}"`;
```

Comment:
It was me suggesting `Your loan on`, but re-reading it I am wondering if it is more `Your loan for...`